tests/cloudwatch/test_cloudwatch_probes.py

```diff
@@ -54,6 +54,49 @@ def test_cloudwatch_get_alarm_state_value_not_found(aws_client):
     client.describe_alarms.assert_called_with(AlarmNames=[alarm_name])
 
 
+@patch('chaosaws.cloudwatch.probes.datetime', autospec=True)
+@patch('chaosaws.cloudwatch.probes.aws_client', autospec=True)
+def test_cloudwatch_get_metric_statistics_dimensions_ok(aws_client,
+                                             mock_datetime):
+    client = MagicMock()
+    aws_client.return_value = client
+    mock_datetime.utcnow.return_value = datetime(2015, 1, 1, 15, 15,
+                                                 tzinfo=timezone.utc)
+
+    namespace = 'AWS/Lambda'
+    metric_name = 'Invocations'
+    dimensions = [{'FunctionName', 'MyFunction'}]
```

Comment:
I think this could do with adhering to:
`Are expected as a list of dictionary objects: [ {‘Name’: ‘DimName1’, ‘Value’: ‘Value1’}, {‘Name’: ‘DimName2’, ‘Value’: ‘Value2’}, … ]` as stated in the docstrings.

I would actually assume this test should *fail* because it is not the expected list format

Reply:
We should probably add verification that the passed in list adheres to the Name & Value keypairs.